docs/src/pages/docs/frequently-asked-questions-page.mdx

```diff
@@ -232,6 +232,29 @@ Metadata attribute using the `label_colors` key.
 }
 ```
 
+### Is there a way to add custom color schemes?
+
+It is possible to override SETUP_EXTRA with COLOR_SCHEMES to register new color schemes
+
+```
+SETUP_EXTRA = {"COLOR_SCHEMES": [
+    {
+        "id": 'myCustomColors',
+        "schemeType": 'Categorical', //Categorical or Sequential
+        "description": '',
+        "label": 'My custom Colors',
+        "colors": ['#6BD3B3', '#6BD3B3', '#6BD3B3', '#66CBE2']
+    }]
+}
+.
+.
+.
+schemes.forEach(scheme => {
+    categoricalSchemeRegistry.registerValue(scheme.id, scheme);
+});
```

Comment:
Is this needed here? The FAQ should probably explain that the change should be done in `superset_config.py`.